### DISCUSSION TITLE
Is there a way to bypass i3wm and pass Alt+Left to an application?

### DISCUSSION THREAD

**Discussion by ajeetdsouza** [i3/i3#5054] [Q&A]
I currently have my `$mod` set to the `Alt` key. Accordingly, my Alt+arrow keys work to navigate between open windows in a workspace, which is great. However, some applications (file browsers, web browsers, code editors) use `Alt+Left/Right` to go forward and back. Some of them (like Google Chrome) don't have configurable keyboard shortcuts.

So, is there a way to pass `Alt+Left` to Chrome etc. without i3wm picking it up? Ideally, I'd like `Alt+[` to pass `Alt-Left` to the current window.

**Comment by Airblader** [comment]
It _might_ work by using `xdotool` to send the key events directly to the Chrome window, but I'm not sure.
